server/cromwell.py

```diff
@@ -0,0 +1,247 @@
+# pylint: disable=unused-variable
```

Comment:
Was that intentionally left?

Reply:
Yes, the way the function definition works inside the add_cromwell_routes throws unused-variable. There's probably a better pattern to use here, but I think it's clear what's happening.

Reply:
Yeah, it's unfortunate that one can't limit the scope of this (because the warning in general is really handy).

Also see https://github.com/PyCQA/pylint/issues/2842.